### ISSUE TITLE
Feature request: Status indicator on statusbar (working/finished)

### ISSUE THREAD

**Issue by ghost** [bdkjones/CodeKit-1-Old#530]
I think a little indicator what or if CodeKit is doing something would be nice. (Example: dropbox statusbar icon)

**Comment by saschamt** [comment]
Agreed. Maybe just show a checkmark sign if the current operation has been completed successfully and an x when there are any errors.
